cmd/down.go

```diff
@@ -18,10 +18,12 @@ package cmd
 
 import (
 	"fmt"
-
```

Comment:
It is common golang style to separate std/non-std import groups: https://github.com/golang/go/wiki/CodeReviewComments#imports